packages/webview/src/App.svelte

```diff
@@ -13,6 +13,7 @@ import DeploymentList from './component/deployments/DeploymentList.svelte';
 import ServicesList from './component/services/ServicesList.svelte';
 import IngressesRoutesList from './component/ingresses-routes/IngressesRoutesList.svelte';
 import PVCsList from './component/pvcs/PVCsList.svelte';
+import JobList from './component/jobs/JobList.svelte';
```

Comment:
Just a naming convention remark: the plural s is missing here compared to the other